src/components/v5/common/ActionSidebar/partials/UserSelect/UserSelect.tsx

```diff
@@ -32,15 +35,21 @@ const UserSelect: FC<UserSelectProps> = ({ name }) => {
     },
   ] = useToggle();
   const { user: userByAddress } = useUserByAddress(field.value);
-  const userDisplayName = userByAddress?.profile?.displayName;
+  const { user: userByName } = useUserByName(field.value);
+  const userDisplayName =
+    userByAddress?.profile?.displayName ||
+    userByName?.profile?.displayName ||
+    field.value;
+  const userWalletAddress =
+    userByAddress?.walletAddress || userByName?.walletAddress || field.value;
```

Comment:
I think it's safe to just use `field.value` here as it will always keep wallet address

Reply:
In which case?